test/baseTest.php

```diff
@@ -106,7 +106,7 @@ public function providerGetLangAndTranslationFile ()
         return array (
             array ("en", "en"),
             array ("fr,fr-fr;q=0.8,en-us;q=0.5,en;q=0.3", "fr"),
-            array ("pt-br,en-us;q=0.7,en;q=0.3", "en"),
+            array ("pt,en-us;q=0.7,en;q=0.3", "en"),
```

Comment:
This tells me COPS algorithm already supports a fallback, we just need a failsafe `pt` language file. What do you think?